src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentPluginApi.java

```diff
@@ -288,12 +292,14 @@ public PaymentModificationResponse execute(@Nullable final Long transactionAmoun
 
     @Override
     public HostedPaymentPageFormDescriptor buildFormDescriptor(final UUID kbAccountId, final Iterable<PluginProperty> customFields, final Iterable<PluginProperty> properties, final CallContext context) throws PaymentPluginApiException {
+        killbillMetrics.markMeter("hpp");
         final Iterable<PluginProperty> mergedProperties = PluginProperties.merge(customFields, properties);
 
         final String amountString = PluginProperties.findPluginPropertyValue(PROPERTY_AMOUNT, mergedProperties);
         Preconditions.checkState(!Strings.isNullOrEmpty(amountString), "amount not specified");
         final BigDecimal amountBD = new BigDecimal(amountString);
         final long amount = amountBD.longValue();
+        killbillMetrics.recordHistogramValue("hpp.amount", amount);
```

Comment:
I disagree. The interesting aspect of these metrics matched with a time series db is exposing a live system status in much easier, faster and straightforward way. That's a different use case from extracting business metrics.